README.md

```diff
@@ -31,7 +31,7 @@ This approach uses an [application template][] to generate a new Rails
 application with Suspenders.
 
 We skip the [default test framework][] in favor of [RSpec][], and [prefer
-PostgreSQL][] as our database.
+PostgreSQL][] as our database, although we do support SQLite.
```

Comment:
I'm being a bit … careful (?) with my README changes here. I want to suggest it's possible, but we're just trying it out, and it's worth evaluating and not that you should necessarily switch away from Postgres.

That said, maybe we need an explicit example?

Reply:
Yeah, this is tricky, and I'm not sure the best approach to take. Suspenders is about limiting the choices we need to make, but I recognize we're in an exploratory phase.

I don't know if we have an explicit example (yet), but I imagine it would relate to avoiding a Platform as a service.

Reply:
I was on the fence about putting a change in at all, but as you can't comment on things you haven't changed in a PR something had to do 😅 .

It's almost like it either needs to be missing, or come with an explanation and I'm not sure what I'd want to do there.